lib/OperationProcessor.ts

```diff
@@ -82,10 +82,7 @@ export default class OperationProcessor {
         return [false, currentDidDocument];
       }
 
-      const originalDidDocument = this.getOriginalDocument(operation);
-      if (originalDidDocument === undefined) {
```

Comment:
Sanity: did you mean to delete the if statement? If so, why?

Reply:
Yes, I believe there is no way we would ever get undefined at this point because the same checks are applied when the operation is created.

Reply:
Please check this claim...